bcdaworker/worker/worker.go

```diff
@@ -151,15 +160,20 @@ func (w *worker) ProcessJob(ctx context.Context, job models.Job, jobArgs models.
 	err = compressFiles(ctx, tempJobPath, stagingPath)
 	if err != nil {
 		logger.Error(err)
+		return err
 	}
 
 	err = createJobKeys(ctx, w.r, jobKeys, job.ID)
 	if err != nil {
 		logger.Error(err)
+		return err
 	}
 
 	// Not critical since we use the job_keys count as the authoritative list of completed jobs.
 	// CompletedJobCount is purely information and can be off.
+	// TODO: Where does this make sense to live?
+	//       Should this just sync the # of job keys into the field since it's the authoritative source?
+	//       Should we remove this field since it's misleading? Is it used anywhere (and is that a problem?)
```

Comment:
This is an open question that should be answered and resolved before merging. I was not sure if que job re-processing should still attempt to...

1. Increment the completed job count (It seems possible this could cause the completed_job_count to be incremented multiple times by the same que job ID)
2. Update the completed job count based on the current job key count (this seems most reliable if we still want this field)
3. Not do this (could cause completed_job_count to be undercounted if a que job is reprocessed)
4. Remove the completed_job_count column (seems best if this column is not accurate and not being used)

Reply:
I think number four sounds like the best option. Based off a quick search through the repo, that count isn't used to verify anything and "it can be off". Because it's inaccurate and we don't use it, getting rid of it would clean up the database and the code overall and make things more understandable.

Reply:
Let's plan for that. Since the column will be removed, I will remove this logic and then we can have a followup ticket to remove the column in the next sprint / after the next deployment (to avoid removing the column before the application is updated.)

Reply:
More changes needed than I thought — splitting into a separate PR here: https://github.com/CMSgov/bcda-app/pull/964